test/test-metrics.c

```diff
@@ -25,6 +25,16 @@
 
 #define UV_NS_TO_MS 1000000
 
+typedef struct {
+  uv_fs_t open_req;
+  uv_fs_t write_req;
+  uv_fs_t close_req;
+} fs_reqs_t;
+
+static uint64_t last_events_count = 0;
```

Comment:
Nit. No need to initialize this
```suggestion
static uint64_t last_events_count;
```